### ISSUE TITLE
survival analysis Kaplan Meier and Cox Regression (Freq and Bayesian)

### ISSUE THREAD

**Issue by cgiannoulis** [jasp-stats/jasp-issues#206]
## Template for bug reports

* JASP version:
* OS name and version:
* Analysis:
* Steps to reproduce: 

## Template for enhancement requests
 
* Enhancement:
* Purpose:

**Comment by TolgaMD** [comment]
Hello. I've recently noticed that we still don't have survival tables or KM analysis/proportional hazards on JASP. I'm VERY hopeful that JASP could solve my never-ending quest for a free and reliable biostatistics package, and I've indeed started sending papers to journals in which I've done analyses using JASP. I believe this is a super-important feature for those using this software for medical research, and I'd be glad if u can prioritize this topic.

**Comment by EJWagenmakers** [comment]
Yes, this is on our agenda, and we'll prioritize this. @jstbcs, maybe something we can discuss...
E.J.

**Comment by srkesler** [comment]
Am having my students in my graduate stats courses use JASP but have to teach them KM and Cox PH in another software....hopefully this is added soon, thanks!

**Comment by ilariagnd** [comment]
This year I will teach basic introductory medical statistics using JASP.  It would be really great to have a tool for survival analysis. Many thanks!

**Comment by RaoVNV** [comment]
Agree with fellow commenters. The only reason I'm not using JASP is because I teach survival analysis, and I only want my students to have to learn one software. Looking forward to the day that JASP can do KM curves and PH regression.

**Comment by FBartos** [comment]
I can completely understand that.

The good thing is that I will start implementing these analyses in a couple of weeks and even though they won't make it into the upcoming release, I'm confident that they will be in the next one!

**Comment by tomtomme** [comment]
any update on this?

**Comment by EJWagenmakers** [comment]
Lots of work has been done behind the scenes!

**Comment by boutinb** [comment]
@FBartos Can you give some updates?

**Comment by FBartos** [comment]
We are about to preprint a paper in which we develop the Bayesian parametric survival models that we will later implement in the module, including the frequentist counterparts.

**Comment by Ermannotrusacani** [comment]
Any news on this? I'll love to use only Jasp for my scientifc paper

**Comment by FBartos** [comment]
The frequentist versions of KM, Cox, and parametric survival models will make it to JASP soon (not the next release, as it will contain only core functionality, but should be able to finish it for the following one). The Bayesian counterparts will unfortunately take a bit longer.

**Comment by nancyluna** [comment]
Any news on this? I'd love to use only Jasp for my scientific paper

**Comment by EJWagenmakers** [comment]
Thanks for the reminder. Frantisek, can you provide an estimate for when this will be included?
Cheers
EJ

**Comment by StevenWallaert** [comment]
We are looking forward to the support of survival analyses so that we can recommend JASP to our students :)

**Comment by EJWagenmakers** [comment]
I am looking forward to it as well. @FBartos should we make a concrete roadmap?

**Comment by FBartos** [comment]
I'm confident that we will have non-parametric and semiparametric survival in the next release.

The non-parametric is mostly done, i.e., see screenshot for the current progress, but there is still more to be done on the Cox PH model. 

![image](https://github.com/jasp-stats/jasp-issues/assets/38475991/0bc61e53-fb48-42ec-93bf-e546476f3d3e)

**Comment by FBartos** [comment]
The non-parametric implementation is gonna be as a beta analysis in the upcoming version, the semiparametric will end up taking  one more release.

**Comment by FBartos** [comment]
Hi, we will have the Cox PH for the next release (it might not have all the functionality, but we will improve it over time as feature suggestions roll in!)

Cheers,
Frantisek
